user-accounts.txt

```diff
@@ -0,0 +1,2 @@
+user1:pass1
```

Comment:
This can be done by storing as username:<encrypted password> and then use `-p` flag in `useradd` command.

Reply:
It is a good idea. 
But only in case of restore. As the password is encrypted.